doc/manuals/admin/database_model.md

```diff
@@ -284,8 +284,8 @@ Fields:
     fields: q, georel, geometry and/or coords (optional)
 -   **count**: the number of notifications sent associated to
     the subscription.   
--   **format**: the format to use to send notification, currently "JSON"
-    meaning JSON notifications in NGSIv1 format.
+-   **format**: the format to use to send notification, possible values are **JSON**
+    (meaning JSON notifications in NGSIv1 format), **normalized**, **keyvalues** and **values**.
```

Comment:
I'd suggest to add at the end of the sentence "... (used in NGSIv2 format)" (to be more comprehensive and have moer "symetry" with the NGSIv1 format description).